src/pages/styles.less

```diff
@@ -83,18 +83,17 @@
   @apply text-green-200;
 }
 
-
 .error {
   @apply text-red-600;
 }
 
 .jsonPretty {
-  height: 600px;
+  height: 500px;
```

Comment:
I think we should revert this size change

Reply:
It's has to the be same height with the container close to it.